digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -224,7 +224,7 @@ object ProjectDAO {
   def getProjectLinksByProjectAndLinkId(projectLinkIds: Iterable[Long], projectId:Long): List[ProjectLink] = {
     val query =
       s"""$projectLinkQueryBase
-                where project_link.id in (${projectLinkIds.mkString(",")}) AND (PROJECT_LINK.PROJECT_ID = $projectId )   order by PROJECT_LINK.ROAD_NUMBER, PROJECT_LINK.ROAD_PART_NUMBER, PROJECT_LINK.END_ADDR_M """
+                where LRM_POSITION.LINK_ID in (${projectLinkIds.mkString(",")}) AND (PROJECT_LINK.PROJECT_ID = $projectId )   order by PROJECT_LINK.ROAD_NUMBER, PROJECT_LINK.ROAD_PART_NUMBER, PROJECT_LINK.END_ADDR_M """
```

Comment:
just curious if this change affected anything, if projectLinkQueryBase already joins project_links and lrm_position

Reply:
ID -> LINK_ID was changed. However we should rename projectLinkIds -> linkIds.